Assignment 5.Rmd

```diff
@@ -66,15 +66,20 @@ Upload the data "intelligent_tutor_new.csv". This is a data set of a differnt sa
 ```{r}
 #Upload new data
 
-D2 <- 
+D2 <- read.csv("intelligent_tutor_new.csv")
 
 #Generate predicted advice using the predict() command for new students based on tree generated from old students
 
-D2$prediction <- 
+D2$prediction <- predict(score_ctree,D2)
 
 ``` 
 ## Part III
 Compare the predicted advice with the actual advice that these students recieved. What is the difference between the observed and predicted results?
+```{r}
+length(which(D2$score==1))/length(D2$score)
+
+length(which(D2$prediction=="no action"))/length(D2$prediction)
+```
```

Comment:
You are on the right track! What does this say about the model? Analyze it in terms of its accuracy rate, as in, specify whether the model overfits or whether it is more generalizable.